Config file config/config.yaml is extended by additional config specified via the command line.
Building DAG of jobs...
Provided cores: 8
Rules claiming more threads will be scaled down.
Conda environments: ignored
Job stats:
job                     count
--------------------  -------
cluster_network             1
prepare_live_network        1
simplify_network            1
solve_network               1
total                       4

Select jobs to execute...

[Tue Apr  2 15:52:19 2024]
rule prepare_live_network:
    input: resources/networks/gen.nc, resources/load_weights.csv, resources/live_data/2024-03-25_12/elexon_bmus.csv
    output: resources/live_data/2024-03-25_12/network.nc
    log: logs/pypsa-lmp/prepare_live_network_2024-03-25_12.log
    jobid: 3
    reason: Code has changed since last execution
    wildcards: date=2024-03-25, period=12
    resources: tmpdir=C:\Users\alyden\AppData\Local\Temp, mem_mb=1500, mem_mib=1431

[Tue Apr  2 15:52:28 2024]
Finished job 3.
1 of 4 steps (25%) done
Select jobs to execute...

[Tue Apr  2 15:52:28 2024]
rule simplify_network:
    input: resources/live_data/2024-03-25_12/network.nc, resources/regions_onshore.geojson, resources/regions_offshore.geojson, data/costs_2020.csv
    output: resources/live_data/2024-03-25_12/network_s.nc, resources/live_data/2024-03-25_12/regions_onshore_s.geojson, resources/live_data/2024-03-25_12/regions_offshore_s.geojson, resources/live_data/2024-03-25_12/busmap_s.csv, resources/live_data/2024-03-25_12/connection_costs_s.csv
    log: logs/pypsa-lmp/simplify_network_2024-03-25_12_s.log
    jobid: 2
    reason: Input files updated by another job: resources/live_data/2024-03-25_12/network.nc
    wildcards: date=2024-03-25, period=12
    resources: tmpdir=C:\Users\alyden\AppData\Local\Temp, mem_mb=1500, mem_mib=1431

[Tue Apr  2 15:52:29 2024]
Error in rule simplify_network:
    jobid: 2
    input: resources/live_data/2024-03-25_12/network.nc, resources/regions_onshore.geojson, resources/regions_offshore.geojson, data/costs_2020.csv
    output: resources/live_data/2024-03-25_12/network_s.nc, resources/live_data/2024-03-25_12/regions_onshore_s.geojson, resources/live_data/2024-03-25_12/regions_offshore_s.geojson, resources/live_data/2024-03-25_12/busmap_s.csv, resources/live_data/2024-03-25_12/connection_costs_s.csv
    log: logs/pypsa-lmp/simplify_network_2024-03-25_12_s.log (check log file(s) for error details)
    conda-env: C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\.snakemake\conda\cf0dbe1799d06fa5b5ff98f73b111298_

RuleException:
CalledProcessError in file C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\Snakefile, line 333:
Command 'C:/Users/alyden/AppData/Local/miniforge3/envs/pypsa-lmp/python.exe "C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\.snakemake\scripts\tmplqhwaqin.simplify_network.py"' returned non-zero exit status 3221225786.
  File "C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\Snakefile", line 333, in __rule_simplify_network
  File "C:\Users\alyden\AppData\Local\miniforge3\envs\pypsa-lmp\Lib\concurrent\futures\thread.py", line 58, in run
Terminating processes on user request, this might take some time.
Complete log: .snakemake\log\2024-04-02T155219.279493.snakemake.log
